client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -51,12 +57,146 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> Analysis,
         isComment -> Comment,
         tagExistsWithId("tone/features") -> Feature,
-        liveBlog -> Live,
-        deadBlog -> Article
+        isLiveBlog -> Live,
+        isDeadBlog -> Article
       )
 
-      val result = predicates.collectFirst { case (predicate, design) if predicate(content) => design }
+      val result = getFromPredicate(content, predicates)
       result.getOrElse(defaultDesignType)
     }
   }
+
+  implicit class RenderingFormat(val content: Content) extends AnyVal {
+
+
+    def design: Design = {
+
+      val defaultDesign: Design = ArticleDesign
+
+      val isPhotoEssay: ContentFilter = content => content.fields.flatMap(_.displayHint).contains("photoessay")
+
+      val isInteractive: ContentFilter = content => content.`type` == ContentType.Interactive
+
+      val predicates: List[(ContentFilter, Design)] = List(
+        tagExistsWithId("artanddesign/series/guardian-print-shop") -> PrintShopDesign,
+        isMedia -> MediaDesign,
+        isReview -> ReviewDesign,
+        tagExistsWithId("tone/analysis") -> AnalysisDesign,
+        tagExistsWithId("tone/comment") -> CommentDesign,
+        tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("tone/features") -> FeatureDesign,
+        tagExistsWithId("tone/recipes") -> RecipeDesign,
+        tagExistsWithId("tone/matchreports") -> MatchReportDesign,
+        tagExistsWithId("tone/interview") -> InterviewDesign,
+        tagExistsWithId("tone/editorials") -> EditorialDesign,
+        tagExistsWithId("tone/quizzes") -> QuizDesign,
+        isInteractive -> InteractiveDesign,
+        isPhotoEssay -> PhotoEssayDesign,
```

Comment:
@oliverlloyd are we keeping Photo Essays as a separate `Design`, or is this temporary until we subsume MultiImage into a normal body element?

Reply:
There is still at least one design aspects of Photo Essays that is unique: coloured captions. So unless we lose that it's here to stay